bitnami/fluentd/templates/aggregator-statefulset.yaml

```diff
@@ -0,0 +1,107 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "fluentd.fullname" . }}
+  labels:
+    {{- include "fluentd.labels" . | nindent 4 }}
+    app.kubernetes.io/component: aggregator
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "fluentd.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+      app.kubernetes.io/component: aggregator
+  serviceName: {{ include "fluentd.fullname" . }}-headless
+  replicas: {{ .Values.aggregator.replicaCount }}
+  updateStrategy:
+    {{- toYaml .Values.aggregator.updateStrategy | nindent 4 }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "fluentd.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        app.kubernetes.io/component: aggregator
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/aggregator-configmap.yaml") . | sha256sum }}
+        {{- if .Values.aggregator.podAnnotations }}
+        {{- toYaml .Values.aggregator.podAnnotations | nindent 8 }}
+        {{- end }}
+    spec:
+      {{- include "fluentd.imagePullSecrets" . | nindent 6 }}
+      {{- if .Values.securityContext.enabled }}
+      securityContext:
+        runAsUser: {{ .Values.securityContext.runAsUser }}
+        fsGroup: {{ .Values.securityContext.fsGroup }}
+        runAsNonRoot: {{ .Values.securityContext.runAsNonRoot }}
+      {{- end }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: {{ include "fluentd.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+            - name: FLUENTD_CONF
+              value: {{ default "fluentd.conf" .Values.aggregator.configFile }}
+            - name: FLUENTD_OPT
+              value: {{ .Values.aggregator.extraArgs }}
+            {{- if .Values.aggregator.extraEnv }}
+            {{- toYaml .Values.aggregator.extraEnv | nindent 12 }}
+            {{- end }}
+          ports:
+            - name: http
+              containerPort: 9880
+              protocol: TCP
+            - name: tcp
+              containerPort: {{ .Values.aggregator.port }}
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /fluentd.healthcheck?json=%7B%22ping%22%3A+%22pong%22%7D
+              port: 9880
+            initialDelaySeconds: 5
+            timeoutSeconds: 1
```

Comment:
`livenessProbe` should be optional (`enabled=true`), also the `initialDelaySeconds`, `timeoutSeconds`, etc should be parametrized. See other charts.

Reply:
Missing readiness probes?